src/super_gradients/training/utils/segmentation_utils.py

```diff
@@ -203,6 +209,67 @@ def target_to_binary_edge(target: torch.Tensor, num_classes: int, kernel_size: i
         considered as edge pixel flatten value is 1. If set as `False` the output tensor shape is [B, C, H, W], else
         [B, 1, H, W]. Default is `True`.
     :return: one_hot edge torch.Tensor.
+
+    Parameters
+    ----------
+    ignore_index
```

Comment:
We use `param PARAM NAME: PARAM DESCRIPTION` docstrings format. Please update the docstrings, otherwise the documentation on the website may break

Reply:
This is not my change and I wasn't working on this function. I can change it if you would like me to do it.

Reply:
@Yael-Baron yes please fix`ignore_index`

Reply:
Fixed on 6e8a1be55dc60d7f9c3bb9fe618d0e4bb5127df4